app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Tag, Typography, Dropdown, Avatar, Tooltip } from "neetoui";
```

Comment:
Let's keep the imports consistent. We don't need the `@bigbinary` namespace. So add relevant aliases.

Reply:
resolved with #16